### PR TITLE
Add font faces

### PR DESCRIPTION
**Pull request by mdaxtman**
@rgerstenberger nailed the select box bug, added system fonts to `constants/index.js` `FontMap`, as well as finished with the font mapping for all the imported font faces.

### CODE REVIEW

**Comment by rgerstenberger** [comment]
*Aside* For branch names go with `feature-name-of-feature` if feature dev, if bug `bug-name-of-bug` or if it's tech debt or other work `chore-name-of-work-being-done` e.g. `feature-text-font-editing`

File: app/constants/index.js [hunk 1]
```diff
@@ -28,60 +28,237 @@ export const SpringSettings = {
...
+    fontStyle: "normal"
+  },
+  blackItalic: {
+    name: "Black Italic",
+    fontWeight: 900,
+    fontStyle: "italic"
+  }
+};
+
 export const FontMap = {
```
Comment (rgerstenberger) [inline app/constants/index.js]: Let's move FontMap to a folder at the same level as constants and elements and call it `font-settings`. `FontMap from this file would be the only export from `font-settings/index.js` Leave `fontTypes` in `constants` and import it into `font-settings/index.js`

**Comment by rgerstenberger** [comment]
screen shot 2016-06-16 at 1 58 42 pm

I know we haven't solved the width/height thing yet but let's keep the initial drag and drop working properly. Please update the initial width/height in elements based on the new default font settings.

File: app/components/property-editor/editor-components/select.js [hunk 2]
```diff
@@ -84,10 +84,16 @@ export default class Select extends Component {
     };
   }
 
+  componentWillReceiveProps() {
+    this.setState({ selectedOptionIndex: false });
```
Comment (rgerstenberger) [inline app/components/property-editor/editor-components/select.js]: Interesting. This is essentially setting state to default state anytime props change. Are there cases where we'd want to update props but keep the selected index?

**Comment by mdaxtman** [comment]
screen shot 2016-06-17 at 10 31 29 am

@rgerstenberger updated styling. Ready for re-review

**Comment by rgerstenberger** [comment]
@mdaxtman the spacing and padding looks off. Also, the closed select box font size looks to big:
screen shot 2016-06-14 at 3 09 48 pm

**Comment by rgerstenberger** [comment]
And I think the check icon is too small or maybe that's because the font size is too big and making it look small.

**Comment by rgerstenberger** [comment]
LGTM!

**Comment by mdaxtman** [comment]
latest screens:
 
screen shot 2016-06-20 at 1 56 00 pm

screen shot 2016-06-20 at 1 56 28 pm

screen shot 2016-06-20 at 1 56 46 pm

**Pull request merged**